lib/facade_converter.ts

```diff
@@ -283,6 +283,31 @@ export class FacadeConverter extends base.TranspilerBase {
         this.emit(')');
       });
     },
+    'Array.forEach': (c: ts.CallExpression, context: ts.Expression) => {
```

Comment:
I'm ok with this going in, but overall I think this is not too useful - we don't check other potentially incorrect Dart code either, and of course we cannot in the general case (as we'd need to understand Dart types). So if anything I wouldn't expand on this line of code.